docs/Installation.md

```diff
@@ -158,10 +158,20 @@ Step 2 of
 
 #### Log in to MariaDB with the root account you set and create the read-only user (with a password, changed below) and database.
```

Comment:
I'll approve this PR, but in a follow-up can you add to this comment something like:

This read-only user will be used by the tool to read the databases. Any updates to the database will still need to use your root/administrative user.

Reply:
Thanks @pleia2! I have addressed your comment in this PR: https://github.com/openmainframeproject/software-discovery-tool/pull/185.